### PR TITLE
Allows admins to spawn mob-like objects for shenanigans

### PR DESCRIPTION
**Pull request by vuonojenmustaturska**
This is basically extracting the functionality of the animation spell into an admin verb. 

Please excuse the browser.dm code, this is more of a stepping stone towards the more complicated popup needed for custom ERTs.

:cl: Naksu
admin: Admins can now easily spawn mobs that look like objects. Googly eyes optional!
/:cl:

image

Requested by admins

### CODE REVIEW

**Comment by tortellinitony** [comment]
Why not tgui?

**Comment by vuonojenmustaturska** [comment]
Because MSO is working on a replacement and the preferences window the full thing is supposed to look like is not tgui.

**Comment by JJRcop** [comment]
Don't want to derail the pr but I just wanna say thar I'm glad to see TGUI go.

**Comment by vuonojenmustaturska** [comment]
image

togopal requested a ckey picker

**Comment by PKPenguin321** [comment]
this but you can spawn a generic /mob but could put your own sprite/HP/pixel shift/attack damage/attack sound before spawning it would also be a blessing

**Comment by vuonojenmustaturska** [comment]
maybe later, i was planning on a second pass on this after the browser control is a bit further along

File: code/datums/browser.dm [hunk 1]
```diff
@@ -309,6 +309,88 @@
...
+		return
+	if (href_list["task"] == "input")
+		var/setting = href_list["setting"]
+		switch (href_list["type"])
+			if ("datum")
+				settings["mainsettings"][setting]["value"] = pick_closest_path(null, make_types_fancy(typesof(text2path(href_list["path"]))))
+			if ("string")
+				settings["mainsettings"][setting]["value"] = input(user, "Enter new value for [settings["mainsettings"][setting]["desc"]]", "Enter new value for [settings["mainsettings"][setting]["desc"]]")
+			if ("number")
+				settings["mainsettings"][setting]["value"] = input(user, "Enter new value for [settings["mainsettings"][setting]["desc"]]", "Enter new value for [settings["mainsettings"][setting]["desc"]]") as num
```
Comment (optimumtact) [inline code/datums/browser.dm]: shouldn't you use stripped_input

Comment (vuonojenmustaturska) [reply code/datums/browser.dm]: not for the number thing, string maybe but... it's an admin interface, do i really care about them possibly injecting html into their own controls?

Comment (optimumtact) [reply code/datums/browser.dm]: right up until someone tries to reuse the datum for a non admin form

**Pull request merged**